src/constants.js

```diff
@@ -106,3 +106,21 @@ export const TARGET_POPULATIONS = [
   'Children with Special Health Care Needs',
   'Dual-Language Learners',
 ];
+
```

Comment:
not for this ticket obv, but I wonder if we should make a package with all the constants shared between frontend and backend.